alpha-features.md

```diff
@@ -0,0 +1,35 @@
+# Open Service Broker API Alpha Features
+
+## Table of Contents
+  - [Alpha Overview](#overview)
+  - [How alpha features are documented](#how-alpha-features-are-documented)
+  - [Current alpha features](#current-alpha-features)
+
+## Overview
+
+The features described in this document are released at an alpha level.  They
+are are available for use in order for the community to provide feedback and
+are not yet an official part of the spec.
```

Comment:
minor thing but since you spell out "specification" in all other cases in this doc, let's do it here too.